code/modules/surgery/organs/organ.dm

```diff
@@ -127,6 +127,9 @@
 	if(istype(loc,/obj/item/mmi))
 		germ_level = max(0, germ_level - 1) // So a brain can slowly recover from being left out of an MMI
 		return TRUE
+	if(istype(loc, /mob/living/simple_animal/hostile/headslug) || istype(loc, /obj/item/organ/internal/body_egg/changeling_egg))
+		germ_level = 0 // weird stuff might happen, best to be safe
+		return TRUE
```

Comment:
God I just realized how disgusting this whole proc is. This should be handled differently, by setting a preserved trait or something. I don't like this, but the proc doesn't appear to use much CPU at all, so it's probably fine.

Reply:
Yeah its fucking rancid : (